lib/sqlalchemy/sql/schema.py

```diff
@@ -2634,19 +2573,6 @@ def __repr__(self):
         return "DefaultClause(%r, for_update=%r)" % (self.arg, self.for_update)
 
 
-@util.deprecated_cls(
-    "0.6",
-    ":class:`.PassiveDefault` is deprecated and will be removed in a "
-    "future release.  Please refer to :class:`.DefaultClause`.",
-)
-class PassiveDefault(DefaultClause):
```

Comment:
this is producing an error because it's referenced in an import:

File "/home/jenkins/workspace/sqlalchemy_gerrit/b3d9d1a2/.tox/py37-cext-sqlite-postgresql/lib/python3.7/site-packages/sqlalchemy/schema.py", line 52, in <module>
INTERNALERROR>     from .sql.schema import PassiveDefault  # noqa

Reply:
can you try running the tests, get it running and then push again here?